pkgs/development/libraries/qt-5/5.10/default.nix

```diff
@@ -37,7 +37,7 @@ let
   srcs = import ./srcs.nix { inherit fetchurl; inherit mirror; };
 
   patches = {
-    qtbase = [ ./qtbase.patch ];
+    qtbase = [ ./qtbase.patch ] ++ optional stdenv.isDarwin ./qtbase-darwin.patch;
```

Comment:
Does this patch need to be updated from version to version?

Reply:
Probably, this fixes compatibility issues with the opensource version of CoreFoundation.

Reply:
Can that not be upstreamed?

Reply:
We could try, but I expect issues like these to keep creeping in if they don't test against it.

Reply:
I would propose to try :)